src/containers/Article.js

```diff
@@ -53,19 +54,16 @@ const _ = {
 const { actions, reduxStateFields, utils } = twreporterRedux
 const { fetchAFullPost } = actions
 
-let articlePostition = {
-  beginY: 120,
-  endY: 200,
-  percent: 0
-}
-
 const scrollPosition = {
   y: 0
 }
 
+const DESKTOP = 'DESKTOP'
+const MOBILE = 'MOBILE'
```

Comment:
We need to update the `MOBILE`, `TABLET`, and `DESKTOP` as constants in  `getScreenType` of `utils`